src/Carousel.js

```diff
@@ -2,6 +2,7 @@ import classNames from 'classnames';
 import styles from 'dom-helpers/style';
 import transition from 'dom-helpers/transition';
 import React, { cloneElement } from 'react';
+import { elementType } from 'prop-types-extra';
```

Comment:
I believe the newest version of `prop-types` now supports elementType, lets use that instead yeah?

Reply:
sure, maybe we should replace it in the whole project? (I can do it if you want)